plugin/build.gradle

```diff
@@ -369,3 +369,166 @@ tasks.withType(licenseHeaders.class) {
 checkstyle {
     toolVersion = '8.29'
 }
+
+String bwcVersion = "1.3.0.0"
```

Comment:
Add how to run bwc test to [developer guide](https://github.com/opensearch-project/ml-commons/blob/main/DEVELOPER_GUIDE.md) like AD https://github.com/opensearch-project/anomaly-detection/blob/main/DEVELOPER_GUIDE.md